holidays/countries/bangladesh.py

```
@@ -119,68 +137,53 @@ class BGD(Bangladesh):
-    }
+    EID_AL_ADHA_DATES_CONFIRMED_YEARS = (2022, 2026)
 
     # https://www.timeanddate.com/holidays/bangladesh/eid-ul-fitr
     EID_AL_FITR_DATES_CONFIRMED_YEARS = (2022, 2026)
     EID_AL_FITR_DATES = {
-        2022: (MAY, 3),
-        2023: (APR, 22),
         2024: (APR, 10),
-        2025: (MAR, 31),
-        2026: (MAR, 21),
     }
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)